### DISCUSSION TITLE
How to style the clear button inside SearchField?

### DISCUSSION THREAD

**Discussion by Taloqq** [adobe/react-spectrum#6269] [Q&A]
I cant figure it out sorry

**Comment by reidbarber** [comment]
**For the [SearchField](https://react-spectrum.adobe.com/react-aria/SearchField.html) in React Aria Components:**

You can style the Button that is within the SearchField. 

You can target the element with the selector ~`.react-aria-Input .react-aria-Button`~ `.react-aria-SearchField .react-aria-Button`, or use any of the styling methods mentioned in the [Styling](https://react-spectrum.adobe.com/react-aria/styling.html#styling) docs.

**For the [SearchField](https://react-spectrum.adobe.com/react-spectrum/SearchField) in React Spectrum:**

Styling this isn't supported in order to maintain consistency with the design system.

**Reply by Taloqq** [reply]
Selector `.react-aria-Input .react-aria-Button` has no effect on the button: 
![clearButton](https://github.com/adobe/react-spectrum/assets/45929791/bf54a4b4-a6c8-4cc7-af66-6d315dcdded9)
Targeting `.react-aria-SearchField` does not work either.

**Reply by devongovett** [reply]
I think it would be .react-aria-SearchField .react-aria-Button. The button is not inside the input but inside the search field.

**Reply by Taloqq** [reply]
@devongovett That does not work either. The button itself is also invisible in the DOM upon inspecting the input field.

**Reply by snowystinger** [reply]
Have you typed anything into the input? It should only appear when there is text that can be cleared.

**Reply by Taloqq** [reply]
@snowystinger Yes, it only appears when there is input. I am trying to change its style, because blue isn't in this projects color palette. I think the default color should be a more neutral color like black, anyway.

**Reply by snowystinger** [reply]
Can you provide a codesandbox or stackblitz showing the problem?

**Reply by uncvrd** [reply]
I think they are referring to the browser default (like in chrome) that adds an "X" to `type="search"` automatically. To remove it and use your own button, I've added this to my stylesheet. which seems to work.

```
    input[type="search" i]::-webkit-search-cancel-button {
        -webkit-appearance: none;
        appearance: none;
        display: none;
    }
```
